subsys/bluetooth/controller/ll_sw/ull_llcp_cc.c

```diff
@@ -852,6 +853,26 @@ static void lp_cc_st_idle(struct ll_conn *conn, struct proc_ctx *ctx, uint8_t ev
 	}
 }
 
+static void lp_cc_state_wait_ntf_avail(struct ll_conn *conn, struct proc_ctx *ctx, uint8_t evt,
```

Comment:
Same as the `lp_cc_st_wait_notify_cancel` implementation in #61536. These should be aligned.

Reply:
When #61536 gets merged, I'll rebase and align.